documentation/questionnaires/questionnaire.schema.json

```diff
@@ -41,15 +41,15 @@
                     },
                     "lowerLabel": {
                         "type": "string",
-                        "description": "The label for the lower end of the lineer scale"
+                        "description": "The label for the lower end of the linear scale"
                     },
                     "upperLabel": {
                         "type": "string",
-                        "description": "The label for the upper end of the lineer scale"
+                        "description": "The label for the upper end of the linear scale"
                     },
-                    "minumum": {
+                    "minimum": {
```

Comment:
Good catch on the typo. I totally missed that

Reply:
All thanks goes to my ide spell checker